src/App.js

```diff
@@ -32,8 +32,18 @@ const App = () => {
       console.log("vowelsArray:", vowelsArray)
 
       // ACTION ITEM: your Pig Latin logic goes here!
-
-    
+      /*
+Implement useState
+Input: apple
+
+if eachWord is vowels
+Oupt: applway
```

Comment:
Would definitely recommend providing more pseudocode, but this looks good otherwise!